plugins/woocommerce/src/Admin/Features/Blueprint/Init.php

```diff
@@ -125,53 +125,53 @@ public function add_woo_importers( array $importers ) {
 	 * @return array
 	 */
 	public function get_step_groups_for_js() {
-			return array(
-				array(
-					'id'          => 'settings',
-					'description' => __( 'It includes all the items featured in WooCommerce | Settings.', 'woocommerce' ),
-					'label'       => __( 'Settings', 'woocommerce' ),
-					'items'       => array_map(
-						function ( $exporter ) {
-							return array(
-								'id'          => $exporter instanceof HasAlias ? $exporter->get_alias() : $exporter->get_step_name(),
-								'label'       => $exporter->get_label(),
-								'description' => $exporter->get_description(),
-							);
-						},
-						$this->get_woo_exporters()
-					),
-				),
-				array(
-					'id'          => 'plugins',
-					'description' => __( 'It includes all the installed plugins and extensions.', 'woocommerce' ),
-					'label'       => __( 'Plugins and extensions', 'woocommerce' ),
-					'items'       => array_map(
-						function ( $key, $plugin ) {
-							return array(
-								'id'    => $key,
-								'label' => $plugin['Name'],
-							);
-						},
-						array_keys( $this->wp_get_plugins() ),
-						$this->wp_get_plugins()
-					),
+		$all_plugins = $this->wp_get_plugins();
+		$active_plugins = array_intersect_key($all_plugins, array_flip(get_option('active_plugins', array())));
+		$active_theme = $this->wp_get_theme();
+
+		return array(
+			array(
+				'id'          => 'settings',
+				'description' => __( 'It includes all the items featured in WooCommerce | Settings.', 'woocommerce' ),
+				'label'       => __( 'Settings', 'woocommerce' ),
+				'items'       => array_map(
+					function ( $exporter ) {
+						return array(
+							'id'          => $exporter instanceof HasAlias ? $exporter->get_alias() : $exporter->get_step_name(),
+							'label'       => $exporter->get_label(),
+							'description' => $exporter->get_description(),
+						);
+					},
+					$this->get_woo_exporters()
 				),
-				array(
-					'id'          => 'themes',
-					'description' => __( 'It includes all the installed themes.', 'woocommerce' ),
-					'label'       => __( 'Themes', 'woocommerce' ),
-					'items'       => array_map(
-						function ( $key, $theme ) {
-							return array(
-								'id'    => $key,
-								'label' => $theme['Name'],
-							);
-						},
-						array_keys( $this->wp_get_themes() ),
-						$this->wp_get_themes()
-					),
+			),
+			array(
+				'id'          => 'plugins',
+				'description' => __( 'It includes all the installed plugins and extensions.', 'woocommerce' ),
```

Comment:
Should we also update the description?

```suggestion
				'description' => __( 'It includes all the active plugins.', 'woocommerce' ),
```